src/sets.jl

```diff
@@ -438,6 +438,25 @@ Base.isapprox(a::SOS2{T}, b::SOS2{T}; kwargs...) where T = isapprox(a.weights, b
 
 dimension(s::Union{SOS1, SOS2}) = length(s.weights)
 
+"""
+    IndicatorSet{ActiveCond, S <:Union{LessThan,GreaterThan,EqualTo}}
+
+Set of (x,y) that satisfy the indicator constraint:
+    y ∈ 𝔹, y = ActiveCond ==> a^T x ∈ S
+
+`S` can be one of `LessThan`, `GreaterThan` or `EqualTo`.
+`ActiveCond` is either true or false.
```

Comment:
For readability, I prefer the enum that I proposed over bools. `IndicatorSet{ZERO_IMPLIES_SATISFIED}` is super clear, while `IndicatorSet{false}` would send me looking to the definition of what `false` means.